frontend/src/tbpro/elements/NoticeBar.vue

```diff
@@ -51,6 +51,7 @@ const isError = props.type === 'error';
   font-size: 0.8125rem;
   font-weight: 700;
   line-height: 1;
+  white-space: pre;
```

Comment:
For our sweet newline character in the localization message

Reply:
Ah, it is intended. Thanks.